.circleci/config.yml

```diff
@@ -3,29 +3,19 @@ version: 2
 jobs:
   deploy:
     docker:
-      - image: google/cloud-sdk:latest
+      - image: publictreemap/public-tree-map-data-pipeline:0.0.1
```

Comment:
Worth noting that making changes to the CI are now harder. Instead of editing a single file, you now may need to edit a separate dockerfile and publish a new image to dockerhub. Not to mention that you now also need a dockerhub account. Just thoughts.

Reply:
thank you @greysonp, you raise important concerns

Reply:
Yes! This is not ideal. I 100% agree with you on that. This would be an issue when the environment changes (either a different version of python, node, or gsutils) and would require a new version for the docker image. This is something to watch out for in pull requests.

See comments below for more details on what I'm currently doing to make it simpler. If you feel like it is good enough or good enough step in the right direction, then it's ready to go.

I'm looking into github actions as a replacement for circleci to fix the above, but we shall see. There is also many many other options such as travis, git labs, azure, bamboo, and probably a few more.

One thing we could do before we decide to cutover to another build tool is try it for a while in parallel for a month or so.

Keep in mind, if you make a change with this branch that would cause a build to fail, it can be checked before you push (manually). I added a make target to run circleci builds locally (which will fail at the google storage upload step due to no google creds locally). I'd like to integrate some sort of automated testing (unit tests and possibly integration tests) in the future to make sure that the build is always good before a pr is merged.

I kind of feel like this is a step in the right direction in terms of making it easy to on-board, but sadly it complicates things due to circleci's constraints (namely no volumes without changing the machine you run on). It also complicates things in terms of using a single docker image using node, python, and google utils requirements.